inference-engine/include/ie_iextension.h

```diff
@@ -349,22 +248,24 @@ class INFERENCE_ENGINE_API_CLASS(IExtension) : public IShapeInferExtension {
         (void)implType;
         return nullptr;
     }
-};
 
-IE_SUPPRESS_DEPRECATED_END_WIN
+    /**
+     * @brief Cleans resources up
+     */
+    virtual void Unload() noexcept = 0;
```

Comment:
It is off topic. But IE doesn't use `Unload()` function, maybe we should refactor this API.

Reply:
ok, in future